testsuite/features/step_definitions/cobbler_steps.rb

```diff
@@ -288,6 +288,8 @@
   else
     $server.run('systemctl restart cobblerd')
   end
+  # give cobbler a second to come up
+  sleep 1
```

Comment:
I don't like this approach.
We have a step that checks if a service is active, we should use it, instead of guessing a period of time that will differ depending the environment or how the machine is under load.